src/wallet/walletdb.h

```diff
@@ -47,8 +47,12 @@ class CHDChain
 public:
     uint32_t nExternalChainCounter;
     CKeyID masterKeyID; //!< master key hash160
+    CExtKey customMasterKey;
+
+    static const int VERSION_BASE = 1;
+    static const int VERSION_WITH_CUSTOM_MASTER_KEY = 2;
+    static const int CURRENT_VERSION = VERSION_WITH_CUSTOM_MASTER_KEY;
```

Comment:
Why add a new version? Why not just overwrite more akin to the way #11085 works?